libraries/service_helpers.rb

```diff
@@ -0,0 +1,58 @@
+module CernerSplunk
+  # Mixin Helper methods for the Splunk Service resource
+  module ServiceHelpers
+    include ResourceHelpers
+    require 'win32/service' if RUBY_PLATFORM =~ /mswin|mingw|windows/
+
+    # Reports if the Windows service for the current package exists
+    # @return [Boolean]
+    def service_exists
```

Comment:
31c954aa1eaaa27c264d333d219f0eb51ad93f64
I didn't change service_running, maybe I can revisit that when I do test refactoring. That's a big maybe though because I suspect this error is occuring because I'm stubbing on Chef::Resource which is very early and it's trying to do some kind of Ruby magic to it and it's just not taking it (hence the question mark in the middle of the method), and whether or not I need to stub Resource or Provider is still a mystery to me, but I know that only one or the other works depending on where it gets called in rspec (first load or after other tests have run), or neither (chef run already happened).